not-supported/remotedesktop.sh

```diff
@@ -329,34 +320,6 @@ We will need to add a 3rd party repository to install it which can set your serv
     *"Nautilus"*)
         install_remove_packet nautilus Nautilus
     ;;&
-    *"OnlyOffice"*)
-        SUBTITLE="OnlyOffice"
-        if is_this_installed onlyoffice-desktopeditors
-        then
-            print_text_in_color "$ICyan" "Uninstalling $SUBTITLE"
-            apt-get purge onlyoffice-desktopeditors -y
-            apt-get autoremove -y
-            rm -f /etc/apt/sources.list.d/onlyoffice-desktopeditors.list
-            apt-get update -q4 & spinner_loading
-            print_text_in_color "$ICyan" "$SUBTITLE was successfully uninstalled."
-        else
-            msg_box "OnlyOffice Desktop Editors are open source but not existing in the Ubuntu repositories.
-Hence, we will add a 3rd-party repository to your server \
-to be able to install and update OnlyOffice Desktop Editors using the apt packet manager.
-This can set your server under risk, though!" "$SUBTITLE"
-            if yesno_box_yes "Do you want to install OnlyOffice Desktop Editors nonetheless?" "$SUBTITLE"
-            then
-                print_text_in_color "$ICyan" "Installing $SUBTITLE"
-                apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys CB2DE8E5
```

Comment:
maybe adding the key in recommended way via add_trusted_key_and_repo can fix this.

Reply:
You should use that function no matter what. The behavior is updated in 22.04, hence the function.